### DISCUSSION TITLE
How to center raw content without centering text in Typst?

### DISCUSSION THREAD

**Discussion by ayoubelmhamdi** [typst/typst#1341] [Q&A]
I'm trying to figure out how to center some `raw` content (such as code or math) without centering the text around it. For example, I want to move the block to center

![rem1](https://github.com/typst/typst/assets/34358145/e0835f75-fedc-49eb-8e82-6dab38014336)

# UPDATE 1
```typst
  show raw: it => {
    set block(inset: 5pt, fill: luma(240))
    pad(0.5em, it)
  }
```

**Comment by Enivex** [comment]
Why not just replace left with center?

**Comment by ayoubelmhamdi** [comment]
`set align(left)` just align the content inside block

**Reply by Enivex** [reply]
Try replacing left with center instead. The content inside the block will still be aligned to the left.

**Comment by ayoubelmhamdi** [comment]
Yes, that’s correct. However, I am currently using `Typst` version 0.2.0. The centering feature works fine in `Typst` version 0.4.0.

 Thank you so much for your help.
